helpers/http.go

```diff
@@ -13,20 +13,26 @@ func ServePageAssets(router *gin.Engine) {
 	if CheckIfFileExists("./public/index.html") {
 		router.Use(static.Serve("/assets", static.LocalFile("./public/assets", true)))
 	} else {
-		router.Use(static.Serve("/assets", static.LocalFile("./views/assets", true)))
+		router.Use(static.Serve("/assets", static.LocalFile("./views/assets", false)))
 		assetsPerPage(router)
 	}
 }
 
 func assetsPerPage(router *gin.Engine) {
-	assetsDir, err := os.ReadDir("./views")
+	views, err := os.ReadDir("./views")
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	for _, directory := range assetsDir {
+	for _, directory := range views {
 		if directory.IsDir() && strings.Contains(directory.Name(), "_page") {
-			router.Use(static.Serve("/"+directory.Name()+"/assets", static.LocalFile("./views/"+directory.Name()+"/assets", true)))
+			assetsPath := "./views/" + directory.Name() + "/assets"
+			if !CheckIfFileExists(assetsPath) {
+				continue
+			}
+
+			// Serve page-specific assets
+			router.Use(static.Serve("/"+directory.Name()+"/assets", static.LocalFile(assetsPath, false)))
```

Comment:
I guess here we can get out of negative if condition, which is always better for readability.
```suggestion
			if CheckIfFileExists(assetsPath) {
				// Serve page-specific assets
			router.Use(static.Serve("/"+directory.Name()+"/assets", static.LocalFile(assetsPath, false)))
			}
```